lib/addons/helm-addon/index.ts

```diff
@@ -23,4 +25,20 @@ export abstract class HelmAddOn implements spi.ClusterAddOn {
         const chart = {...this.props, ...{ values, wait, timeout, createNamespace} };
         return kubectlProvider.addHelmChart(chart);
     }
-}
\ No newline at end of file
+}
+//below is my addon approach method for when I get too it. Leaving for later
+function generalAddonConstraintsValidation(props: HelmAddOnProps){
+    try{
+        const test = z.string().max(addonConstraints.addonNameMaxLength);
+        console.log(props.name + "-=-=-=-=-=")
+        test.parse(props.name);
```

Comment:
Please address the general GitHun actions warnings.

Reply:
I have gone through them now I believe.